ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh

```diff
@@ -19,13 +19,19 @@ fi
 
 # Do not change the default family type without consulting with cloud financial operations as their may
 # be active savings plans targeting this machine class.
-master_type=e2-standard-4
+master_type=""
+master_type_suffix=-standard-4
 if [[ "${SIZE_VARIANT}" == "xlarge" ]]; then
-  master_type=e2-standard-32
+  master_type_suffix=-standard-32
 elif [[ "${SIZE_VARIANT}" == "large" ]]; then
-  master_type=e2-standard-16
+  master_type_suffix=-standard-16
 elif [[ "${SIZE_VARIANT}" == "compact" ]]; then
-  master_type=e2-standard-8
+  master_type_suffix=-standard-8
+fi
+if [ "${OCP_ARCH}" = "amd64" ]; then
+  master_type=e2${master_type_suffix}
```

Comment:
suggest to use double quote - `"e2${master_type_suffix}"`

Reply:
the installer pr has merged, and we can install arm cluster on gcp now.  wait for your whole comments, and will fix them together